app/src/main/java/com/example/vaccinespotter/worker/RetrofitManager.java

```diff
@@ -0,0 +1,57 @@
+package com.example.vaccinespotter.worker;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.example.vaccinespotter.notifications.VaccineNotificationManager;
+import com.example.vaccinespotter.apiinterface.VaccineSlots;
+import com.example.vaccinespotter.models.Center;
+import com.example.vaccinespotter.models.Centers;
+import com.example.vaccinespotter.models.NotificationModel;
+import com.example.vaccinespotter.models.Session;
+import java.util.List;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitManager {
+    private final String TAG = "RetrofitManager";
+    private static final int age18 = 18;
+    private static final int districtId = 312;

```

Comment:
If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere.